glide.yaml

```diff
@@ -11,7 +11,7 @@ import:
 - package: github.com/gorilla/mux
   version: ~1.1.0
 - package: github.com/prometheus/client_golang
-  version: ~0.8.0
+  version: master
```

Comment:
Should we maybe stick to a concrete tag? IMO we should try to be a bit careful with external deps on master.